dockerfiles/mlrun-api/Dockerfile

```diff
@@ -46,6 +46,9 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 
 ENV UV_SYSTEM_PYTHON=true UV_LINK_MODE=copy
 
+ARG MLRUN_PIP_VERSION=25.0
+RUN python -m pip install --upgrade pip~=${MLRUN_PIP_VERSION}
```

Comment:
```suggestion
```

There's no use for pip in this Dockerfile, no need to add it

Reply:
correct that there is no use, 
*but* the default image comes *with pip* which is old, so better of upgrading it

Reply:
Yep, I see (thought about it afterward).

Then why not:

1. Uninstall pip in this Docker image
2. Upgrade pip in the base image - `gcr.io/iguazio/python:3.9-slim`

Reply:
1. optional
2. is mirror-ed by `python:3.9-slim`